### ISSUE TITLE
Accuracy repro extraction, constants in graph are not preserved exactly

### ISSUE THREAD

**Issue by ezyang** [pytorch/pytorch#100468]
### 🐛 Describe the bug

If the repro graph contains a constant (e.g., due to use of torch.tensor), this constant is not serialized faithfully when we extract repros. I plan on fixing it so that if the constant is not too big (<= 4) we'll just print it into the graph, but ideally we would use the same ContentStore mechanism to store these constants exactly.

Comment on this issue if this is affecting you.

### Versions

master

cc @msaroufim @bdhirsh @anijain2305 @zou3519 @chauhang @wconstab @ngimel @soumith

**Comment by tugsbayasgalan** [comment]
@ezyang any updates on this issue?

**Comment by ezyang** [comment]
yes
